packages/theme-provider/package.json

```diff
@@ -12,7 +12,9 @@
     "@emotion/core": "^10.0.0",
     "@theme-ui/color-modes": "^0.3.1",
     "@theme-ui/core": "^0.3.1",
-    "@theme-ui/mdx": "^0.3.0"
+    "@theme-ui/css": "^0.3.1",
```

Comment:
Thanks for looking into this, but since, for example, `@theme-ui/core` depends on `@theme-ui/css`, I don't think these should be added as dependencies to this package

Reply:
but you are importing that package inside theme-provider hence yarn complaining about you not listing it it dependencies.
https://github.com/system-ui/theme-ui/blob/e3a24cee136553ee62ea777ca99037612b5f5ff1/packages/theme-provider/src/index.js#L2